### DISCUSSION TITLE
IOS apps not working nest, ecobee and textnow

### DISCUSSION THREAD

**Discussion by tanzeelareeb** [AdguardTeam/AdGuardHome#2740] [Q&A]
IOS apps not working nest, ecobee and textnow. I have the latest version and I am using it with home assistant. I tried all the available processes to avoid that but no effect.

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [ -] I am running the latest version
- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed

### Issue Details

* **Version of AdGuard Home server:** Version: 0.104.3
* **How did you install AdGuard Home:** Homeasistant 
* **How did you setup DNS configuration:** Router
* **If it's a router or IoT, please write device model:** Netgear R8000
* **Operating system and version:** Homeasistant OS 5.10, core-2021.1.5, supervisor-2021.01.7

### Screenshots

<details><summary>Screenshot:</summary>

</details>

**Comment by ameshkov** [comment]
Do they work if you "Disable protection" in AdGuard Home? In this case, nothing will be blocked and we'll see if this is a bug or just a filters issue.

**Comment by tanzeelareeb** [comment]
When I disable the filter apps starts working again

**Comment by ameshkov** [comment]
Check the query log to see what queries are being blocked, try unblocking them one by one.
